README.md

```diff
@@ -248,3 +248,40 @@ To compare all mails on the list against upstream:
 [1]: https://public-inbox.org/README.html
 [2]: https://github.com/xai/nntp2mbox
 [3]: https://git.kernel.org/pub/scm/public-inbox/
+
+PaStA and Patchwork
+--------------------
+The results of PaStA's analyses can be used by [Patchwork](http://jk.ozlabs.org/projects/patchwork/).
+
+## Setting up PaStA and Patchwork
+Assuming a working setup of PaStA already exists, here are the steps necessary for Patchwork integration
+
+1. Install Patchwork on your system, following the guidelines in Patchwork's [documentation](https://patchwork.readthedocs.io/en/latest/development/installation/)
+
+2. Start a shell inside Patchwork's docker container with `docker-compose run --rm web --shell`
+
```

Comment:
Could you please add two more words how to:
  - create a test project in patchwork
  - import parts of initial archive
  - import the rest of the archive

Reply:
Sure